notifier.py

```diff
@@ -174,42 +156,47 @@ def nvidia_get(url, api_url):
         alert(url)
 
 
+with open('sites.json', 'r') as f:
+    sites = json.load(f)
+
+
 def main():
     search_count = 0
     while True:
         now = datetime.now()
         current_time = now.strftime("%H:%M:%S")
         print("Starting search {} at {}".format(search_count, current_time))
         search_count += 1
-        for url, info in url_keywords.items():
-            print("\tChecking {}...".format(info[3]))
-
-            try:
-                if info[2] == GET_SELENIUM:
-                    if not USE_SELENIUM:
+        for site in sites:
+            if site['enabled']:
+                print("\tChecking {}...".format(site['name']))
+
+                try:
+                    if site['method'] == "GET_SELENIUM":
```

Comment:
This is minor, but try to avoid direct string comparisons where possible. Maybe throw an enum somewhere.

Reply:
updated